docs/subnamespaces.md

```diff
@@ -30,7 +30,7 @@ metadata:
     team: foo
 ```
 
-Accurate only propagates labels/annotations that have been configured in that respect via the `labelKeys` and `annotationKeys` parameters in `config.yaml`. This prevents the propagation of labels/annotations that were not meant to do so. Accurate currently does not delete previously propagated labels when deleted from the parent namespace to prevent unintended deletions. Users are expected to manually delete labels/annotations that are no longer needed.
+Accurate only propagates labels/annotations that have been configured in that respect via the `labelKeys` and `annotationKeys` parameters in `config.yaml`. This prevents the propagation of labels/annotations that were not meant to do so.
```

Comment:
This PR includes a change in specifications.
Could you please tell me why you are changing this behavior?

Reply:
@yamatcha 
This is the very reason why @erikgb is making this PR.
Without SSA, it was difficult to remove labels and annotations safely.
Now with SSA, removing them is a safe operation.

Reply:
Exactly! @ymmt2005 is correct that this is the main motivation for this PR. There is yet no 1. class support for SSA in controller-runtime, but I have been involved in migrating several operators to SSA using the same approach as in this PR. So I am pretty sure it works, and it's also properly tested now IMO. The main concern with SSA migration is the risk of getting never-ending reconcile loops, but I have added new tests for this.

We would be extremely happy if this PR could eventually get merged and released. https://github.com/cybozu-go/accurate/issues/98 is a semi-blocker for our further rollout of Accurate in our clusters.

Reply:
@erikgb 
Thank you for the detailed description! 
I missed issue refered  in PR description.